deps/rabbit/src/rabbit_quorum_queue.erl

```diff
@@ -1783,6 +1784,15 @@ get_nodes(Q) when ?is_amqqueue(Q) ->
     #{nodes := Nodes} = amqqueue:get_type_state(Q),
     Nodes.
 
+get_voters(Q) when ?amqqueue_is_quorum(Q) ->
+    {_, Leader} = amqqueue:get_pid(Q),
+    %% FIXME replace with proper API pending rabbitmq/ra#411
+    case  ra_server_proc:state_query(Leader, voters, 1000) of
```

Comment:
Timeouts this low are guaranteed to produce false positives in our experience. Please bump this to 5s.